website/docs/docs/dbt-cloud/dbt-cloud-enterprise/enterprise-permissions.md

```diff
@@ -3,43 +3,125 @@ title: "Enterprise Permissions"
-
-
-## Project level permission sets
-
-The following permissions may be assigned to groups at the project-level.
-
-- Admin:
-  - Can create, read, update, and delete all objects in assigned projects
-  - Can add groups with any permission to assigned projects
-  - Implies GitAdmin, DatabaseAdmin, and TeamAdmin permissions sets
-  - Requires user to have a developer user license 
-- GitAdmin:
-  - Can create, read, update, and delete the git repo in assigned projects
-  - Requires user to have a developer user license 
-- DatabaseAdmin:
-  - Can create, read, update, and delete the database connection in assigned projects
-  - Can create, edit, update, and delete deployment credentials in assigned projects
-  - Requires user to have a developer user license 
-- TeamAdmin:
-  - Can create, read, update, and delete group membership in the project
-  - Can only groups to assigned projects with Developer, Stakeholder, or TeamAdmin permissions
-  - Requires user to have a developer user license 
-- Developer:
-  - Can access the IDE, configure personal developer credentials, and manage job definitions in assigned projects
-  - Requires user to have a developer user license 
-- Stakeholder:
-  - Can access the documentation and source freshness reports for assigned projects
-  - User can have a read-only or a developer user license
-
-## Account-wide permissions:
-
-The following permissions are assigned outside the scope of any particular project, and apply to the entire account.
-
-- AccountAdmin
-  - Create Projects
-  - Create groups, assign SSO mappings between SSO groups and dbt Cloud groups
-  - Manage notifications
-  - Manage account-wide build artifacts
-  - Requires user to have a developer user license 
+The following permission sets are available for assignment for dbt Cloud Enterprise.
+Please contact your Account Manager to implement the permissions defined here.
+See [Seats and Users](cloud-seats-and-users) for more information on user licenses.
+
+## Permission Sets
+
+The following permission sets are available for assignment in dbt Cloud Enterprise accounts.
+
+### Account Admin
+- **Authority:** Account-wide
+- **License restrictions:** must have a developer license
+
+Account Admins have unrestricted access to dbt Cloud accounts. Users with Account Admin permissions can:
+- Create, delete and modify all projects in an account
+- Create, delete, and modify Repositories
+- Create, delete, and modify Connections
+- Create, delete, and modify Environments
+- Create, delete, and modify Jobs
+- Create, delete, and modify Groups
+- Create, delete, and modify Group Memberships
+- Manage notification settings
+- Manage account-level artifacts
+- View and modify Account Settings
+- Use the IDE
+- Run and cancel jobs
+
+### Admin
+- **Authority:** Account-wide, or project-scoped
```

Comment:
I don't know if it's just me but the concept of `Account-wide` sounds confusing especially because it's really saying you can select `all projects`.

Reply:
Ok - i changed:
**Authority:** Account-wide, or project-scoped

to:
 
Account Admin:
- **Has permissions on:** Authorized projects, account-level settings

Every other permissionset:
 - **Has permissions on:** Authorized projects

Reply:
I love it!